server/server.go

```diff
@@ -42,6 +42,7 @@ type Config struct {
 	GRPCListenPort   int
 
 	RegisterInstrumentation bool
+	TruncatedServerLogging  bool
```

Comment:
could name this after what it does, e.g. `LogRequest` or `IncludeRequestInLog` or something like that